platform/api/esx/api.go

```diff
@@ -282,6 +282,32 @@ func (a *API) GetConsoleOutput(name string) (string, error) {
 	return string(buf), nil
 }
 
+func (a *API) CleanupFiles(name string) error {
```

Comment:
For safety purposes it's probably worth implementing the check if it's ever used outside of the `platform/machine/cluster` implementation.

I'll rename it to `CleanupDevice` as it requires the context of a VM